alacritty.yml

```diff
@@ -468,27 +478,25 @@
 #
 # - `action`: Execute a predefined action
 #
-#   - Copy
-#   - Paste
-#   - PasteSelection
-#   - IncreaseFontSize
-#   - DecreaseFontSize
-#   - ResetFontSize
-#   - ScrollPageUp
-#   - ScrollPageDown
-#   - ScrollLineUp
-#   - ScrollLineDown
-#   - ScrollToTop
-#   - ScrollToBottom
 #   - ClearHistory
-#   - Hide
-#   - Minimize
-#   - Quit
-#   - ToggleFullscreen
-#   - SpawnNewInstance
 #   - ClearLogNotice
-#   - ReceiveChar
-#   - None
+#   - SpawnNewInstance
+#   - None, ReceiveChar
+#   - Hide, Minimize, Quit, ToggleFullscreen
+#   - Copy, Paste, PasteSelection
+#   - IncreaseFontSize, DecreaseFontSize, ResetFontSize
+#   - ScrollPageUp, ScrollPageDown, ScrollHalfPageUp, ScrollHalfPageDown,
+#     ScrollLineUp, ScrollLineDown, ScrollToTop, ScrollToBottom
+#   - ToggleKeyboardMode, ToggleNormalSelection, ToggleLineSelection,
+#     ToggleBlockSelection, ClearSelection, KeyboardMotionLaunchUrl,
+#     KeyboardMotionUp, KeyboardMotionDown, KeyboardMotionLeft,
+#     KeyboardMotionRight, KeyboardMotionStart, KeyboardMotionEnd,
+#     KeyboardMotionHigh, KeyboardMotionMiddle, KeyboardMotionLow,
+#     KeyboardMotionSemanticLeft, KeyboardMotionSemanticRight,
+#     KeyboardMotionSemanticLeftEnd, KeyboardMotionSemanticRightEnd,
+#     KeyboardMotionWordRight, KeyboardMotionWordLeft,
+#     KeyboardMotionWordRightEnd, KeyboardMotionWordLeftEnd,
+#     KeyboardMotionBracket
```

Comment:
This should be grouped better at least. I'm finding it hard to read (granted it's 1 AM and I'm sleepy now).